.github/workflows/ci-crossdock.yml

```diff
@@ -17,10 +17,6 @@ jobs:
       with:
         submodules: true
 
-    - uses: actions/setup-python@v2
-      with:
-        python-version: 2.7
```

Comment:
I assume we don't want to remove the step but to upgrade it to 3.x

Reply:
Upgraded to 3.6